src/js/components/Field/Field.tsx

```diff
@@ -0,0 +1,90 @@
+/**
+ * @name Field
+ *
+ * @description
+ *  A field group component is a wrapper for grouping the input with label and messages.  See the teamsnap patterns
+ *  library for more information. https://teamsnap-ui-patterns.netlify.com/patterns/components/field-group.html
+ *
+ * @example
+ *  <FieldGroup name='example' label='Check Me' isInline>
+ *    Field Child Data
+ *  </FieldGroup>
+ *
+ */
+
+import * as React from "react";
+import * as PropTypes from "prop-types";
+import { FieldGroup } from "../FieldGroup";
+import { FieldLabel } from "../FieldLabel";
+import { FieldMessage } from "../FieldMessage";
+import { Input } from "../Input";
+import { Status, Size } from "../../types";
+
+const fieldPropTypes = {
+  type: PropTypes.oneOf(["toggle", "select", "input"]).isRequired,
```

Comment:
How does this work with Checkbox/Radio... or is that coming later?

Reply:
Coming soon!™